docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: linux-vm-${NAME}
+  spec:
+    template:
+      metadata:
+      labels:
```

Comment:
What is the role of labels here? Is the OfflineVirtualMachine controller supposed to generate them automatically?

Reply:
Will remove all the groups as suggested

Reply:
Currently OfflineVirtualMachine does not require the labels. That is the reason of my curiosity :)

Reply:
@petrkotas  well it was picked up from the Offline virtual machine doc :)  can be removed if not needed

Reply:
BTW, I was following openshift guidelines here: https://docs.openshift.org/latest/dev_guide/templates.html#writing-labels
not sure how those labels (not the metadata.labels) with enter into play.